README.md

```diff
@@ -31,6 +34,28 @@ installing a more recent gem.  Building and installing signalfx from source will
 
 ## Usage
 
+
+### Configuring your endpoints
+
+In order to send your data to the correct realm, you may need to configure your 
+endpoints. If no endpoints are set manually, this library uses the ``us0`` realm by default. 
+If you are not in this realm, you will need to explicitly set the
+endpoint config options below. To determine if you are in a different realm and need to
+explicitly set the endpoints, check your profile page in the SignalFx 
+web application. You will also need to specify an access token when requesting
+one of those clients. For the ingest client, you need to specify your
+organization access token (which can be obtained from the
+SignalFx organization you want to report data into). For the REST API,
+you must use your user access token. For the SignalFlow client, either an
+organization access token or a user access token may be used. For more
+information on access tokens, see the API's [authentication documentation](https://developers.signalfx.com/basics/authentication.html).
+
+```ruby
+ingest_endpoint https://ingest.{REALM}.signalfx.com
+api_endpoint https://api.{REALM}.signalfx.com
+stream_endpoint https://stream.{REALM}.signalfx.com
+```
+
 ### API access token
 
 To use this library, you need a SignalFx API access token, which can be obtained from the SignalFx organization you want to report data into.
```

Comment:
Terminology needs to be consistent (even though it's confusing). Do you mean org access token? (supports ingest and all APIs except member management).